docs/image/skymaps.rst

```diff
@@ -63,3 +63,72 @@ The sky map can be easily displayed with an image viewer, by calling ``skymap.sh
         counts = SkyMap.read(filename)
         counts.name = 'Counts'
         counts.show()
+
+
+Working with the cutout and paste method
```

Comment:
I have a few docs suggestions.

Make the title shorter "Cutout and paste" so that it's nicer in the navigation side bar.

Add a label in front of high-level docs sections, in this case e.g.
```
.. _skymap-cutpaste:

Cutout and paste
----------------
```

And then in the docstring of cutout and paste, link to this high-level docs section via
```
See :ref:`skymap-cutpaste` for information how to cut and paste sky images.
```
and then don't add another example to those docstrings.